system-addon/content-src/components/Search/Search.jsx

```diff
@@ -22,6 +22,13 @@ class Search extends React.Component {
   onClick(event) {
     window.gContentSearchController.search(event);
   }
+  componentDidMount() {
+    // Focus the search box if we are on about:home
+    if (!IS_NEWTAB) {
+      const searchBox = document.getElementById("newtab-search-text");
+      if (searchBox) { searchBox.focus(); }
```

Comment:
The tricky part here is if we mount multiple times for unknown reasons.. But this might be fine for now.

Reply:
I think I can request browser focus from here by dispatching an action?

Reply:
Ah actually that might work where we assume about:home is not preloaded. Also, any reason not to do this from `onInputMount`?

Reply:
about:home is currently never preloaded, though it's likely to be pre-rendered.